### PR TITLE
Fixes BoxStation Stacking Machine, while adding a Stacking Machine in CereStation

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
This PR fixes Cyberiad's stacking machine - It used to be varedited to only accept inputs from the North side (Which is a wall), which causes materials from the recycler (Which comes from the West side) to never get stored in the stacking machine. Now it is varedited to accept inputs from the West side and release any stored mineral sheets on the South side of the stacking machine.

The PR also adds a stacking machine (And the associated console) back into Farragus's Disposal. The layout of the room has thus been changed slightly to accomodate the extra machines.

## Why It's Good For The Game
It is confusing and a bug, and I have gotten questions on Discord about how stacking machines work and why does the one in Cyberiad doesn't work.

Farragus Disposals is also missing a stacking machine, which does not align with other stations's Disposals, and I have confirmed with Sean on Discord that it is supposed to have one.

## Images of changes
Farragus Before:
Screenshot (649)

Farragus After:
Screenshot (650)

Not going to put an image of the Cyberiad change because it is literally just a varedit on StrongDMM - Nothing to see there.

## Testing
Loaded up local server, threw random maintenance objects and metal sheets into recycler, turned on conveyor belt and verified that stacking machine has stored mineral sheets within, then released said mineral sheets and verified that they appeared.

## Changelog
:cl:
add: Added missing stacking machine and stacking machine console to Farragus Disposals
fix: Cyberiad Disposals stacking machine now properly stacks minerals from the recycler
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
image
this can be removed from the changelog.

File: _maps/map_files/cerestation/cerestation.dmm [hunk 1]
```diff
@@ -84287,6 +84307,16 @@
 	icon_state = "neutralfull"
 	},
 /area/station/hallway/primary/port/east)
+"urK" = (
+/obj/machinery/mineral/stacking_machine{
+	stack_amt = 10;
```
Comment (Henri215) [inline _maps/map_files/cerestation/cerestation.dmm:84312]: Any idea why we varedit to limit it to 10? it was the case with the cyberiad one too and you removed it.

Comment (SchrodingersWolf) [reply _maps/map_files/cerestation/cerestation.dmm:84312]: I just copied the one from Cyberiad - I didn't know why it was varedited to only limit it to 10 and I don't think I removed it on Cyberiad.

I'll probably remove the 10 stack limit - The recycler doesn't need more things limiting it really.

**Comment by SchrodingersWolf** [comment]
> image
> 
> this can be removed from the changelog.

Done.

File: _maps/map_files/cerestation/cerestation.dmm (hunk 1 above)
Comment (Henri215) [reply _maps/map_files/cerestation/cerestation.dmm:84312]: The original amount is 50, i always felt like 10 was too small, feel free to remove it?

Comment (SchrodingersWolf) [reply _maps/map_files/cerestation/cerestation.dmm:84312]: Stacking machine on Farragus back to 50.

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by Henri215 — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Comment by S34NW** [comment]
Map queue awaits before merge

**Pull request merged**